sensu/check.go

```diff
@@ -15,17 +15,22 @@ const (
 	CheckStateUnknown  = 3
 )
 
-type GoCheck struct {
-	basePlugin
+// GoCheck is deprecated, use Check
```

Comment:
Is there a standard deprecated comment format so that linters pick it up? I know I've seen my editor yell at me for it before but I'm not sure it's standardized.

Reply:
Good call, I should be using this format: https://github.com/golang/go/wiki/Deprecated